Zend/zend_exceptions.c

```diff
@@ -985,22 +986,28 @@ ZEND_API ZEND_COLD int zend_exception_error(zend_object *ex, int severity) /* {{
 }
 /* }}} */
 
-ZEND_API ZEND_COLD void zend_throw_exception_object(zval *exception) /* {{{ */
+ZEND_API ZEND_COLD void zend_throw_exception_obj(zend_object *exception) /* {{{ */
```

Comment:
I'd just use zend_throw_exception_internal in the one place that needs it, rather than introducing a separate API function.